source/guide_the-lounge.rst

```diff
@@ -0,0 +1,219 @@
+.. author:: roang <https://github.com/Roang-zero1>
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/the-lounge.svg
+      :align: center
+
+#####
+The Lounge
+#####
+
+`The Lounge`_ is an open source IRC web client written in JavaScript and distributed under the MIT License, this self hosted client stays always connected so you never miss out on the most important chats. Another goal of The Lounge is to bring modern chat features such as push notifications, link previews and many more to your IRC chats.
+
+The Lounge is based on the project Shout_ from which it is a community driven fork.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Node.js_ and its package manager npm_
+  * supervisord_
+  * domains_
+
+Prerequisites
+=============
+
+We're using Node.js_ in the stable version 8:
+
+::
+
+ [isabell@stardusts ~]$ uberspace tools version show node
+ Using 'Node.js' version: '8'
+ [isabell@stardusts ~]$
+
+Your The Lounge URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Install thelounge
+-----------------
+
+Use ``npm`` to install ``thelounge`` globally:
+
+::
+
+ [isabell@stardusts ~]$ npm install --global thelounge
+ [...]
+ + thelounge@2.7.0
+ added 242 packages in 31.627s
+ [isabell@stardusts ~]$
+
+Verify installation
+-------------------
+
+::
+
+ [isabell@stardusts ~]$ thelounge -v
+ v2.7.0
+
+Configuration
+=============
+
+Configure port
+--------------
+
+Since Node.js applications use their own webserver, you need to find a free port and bind your application to it.
+
+.. include:: includes/generate-port.rst
+
+Setup .htaccess
+---------------
+
+.. include:: includes/proxy-rewrite.rst
+
+Create configuration file
+------------------------
+
+Create the configuration directory:
+
+::
+
+ [isabell@stardusts ~]$ mkdir ~/.lounge
+ [isabell@stardusts ~]$
+
+
+To configure The Lounge you need to create the ``~/.lounge/config.js`` file with the following content:
+
+.. code-block:: javascript
+
+ "use strict"
+
+ module.exports = {
+   public: false,
+   host: '127.0.0.1',
+   port: <yourport>,
+   reverseproxy: true
+ };
+
+Alternatively you can download the full default configuration from the project repository:
+
+.. warning:: If you choose this option change the values according to the config provided above.
+
+::
+
+ [isabell@stardusts ~]$ wget https://raw.githubusercontent.com/thelounge/thelounge/master/defaults/config.js > ~/.lounge/config.js
+ [isabell@stardusts ~]$
+
+From now on you can modify with the ``thelounge config`` command.
+
+For a full list of all configuration options see `the official documentation`_.
+
+Creating users
+--------------
+
+Since we do not run a public server we have to add users before we can use The Lounge:
+
+::
+
+ thelounge add isabell
```

Comment:
The `~/.thelounge/users/` directory needs to be created before adding users

```
[nico@stardust ~]$ thelounge add nico
2018-10-09 17:24:33 [ERROR] /home/nico/.lounge/users does not exist.
```

Reply:
The first start of the service will create all necessary directories, so I moved the user management section to the **Finishing installation** section.